--- dist/pasmo.js	2026-05-02 22:48:53.746617759 +0000
+++ pasmo.js	2026-05-02 22:56:03.401915957 +0000
@@ -86,8 +86,21 @@
                 };
             }
         } else {}
-        var out = console.log.bind(console);
-        var err = console.error.bind(console);
+
+        var out;
+        if(moduleArg.log) {
+            out = moduleArg.log;
+        } else {
+            out = console.log.bind(console);
+        }
+
+        var err;
+        if(moduleArg.err) {
+            err = moduleArg.err;
+        } else {
+            err = console.error.bind(console);
+        }
+        
         var wasmBinary;
         var ABORT = false;
         var EXITSTATUS;
